Source/missiles.h

```diff
@@ -97,8 +97,8 @@ struct Missile {
 	const byte *_miAnimData;
 	int _miAnimDelay; // Tick length of each frame in the current animation
 	int _miAnimLen;   // Number of frames in current animation
-	int _miAnimWidth;
-	int _miAnimWidth2;
+	uint16_t _miAnimWidth;
+	int16_t _miAnimWidth2;

```

Comment:
Why does this need to be signed, isn't it just half of the other one? Or is it an offset?

Reply:
Not entirely sure what this represents but it is negative sometimes